### PR TITLE
Remove Transaction#info: [ECR-3793]

### PR DESCRIPTION
**Pull request by dmitry-timofeev**
## Overview

This operation is no longer used by the framework.

---
See: https://jira.bf.local/browse/ECR-3793

### Definition of Done

- [x] There are no TODOs left in the code
- [x] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [x] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [x] Public API has Javadoc
- [x] Method preconditions are checked and documented in the Javadoc of the method
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

### CODE REVIEW

File: exonum-java-binding/fakes/src/main/java/com/exonum/binding/fakes/services/transactions/SetEntryTransaction.java [hunk 1]
```diff
@@ -1,89 +0,0 @@
...
-import com.exonum.binding.core.transaction.TransactionContext;
-import com.google.common.annotations.VisibleForTesting;
-
-/**
- * A transaction whose behaviour can be configured. It's not a mock: it writes a given value
- * into the database in its {@link #execute(TransactionContext)}.
- *
- * <p>Such transaction is supposed to be used in TransactionProxy integration tests.
- */
-public final class SetEntryTransaction implements Transaction {
```
Comment (MakarovS) [inline exonum-java-binding/fakes/src/main/java/com/exonum/binding/fakes/services/transactions/SetEntryTransaction.java]: Why did you remove the whole transaction and not just `info()`?

Comment (dmitry-timofeev) [reply exonum-java-binding/fakes/src/main/java/com/exonum/binding/fakes/services/transactions/SetEntryTransaction.java]: It used to be used in native ITs of Rust TransactionProxy, which are gone. Probably, most of fakes will be gone, but that's after @ipenkin 's tests.
